mender-update/deployments.cpp

```diff
@@ -314,6 +315,93 @@ error::Error PushStatus(
 		});
 }
 
+static const string logs_uri_suffix = "/log";
+
+error::Error PushLogs(
+	const string &deployment_id,
+	LogMessages messages,
+	const string &server_url,
+	http::Client &client,
+	events::EventLoop &loop,
```

Comment:
Should remove this, see https://github.com/mendersoftware/mender/pull/1301.

Reply:
Yup! Too much copy pasta going all the way from the original `GetJWTToken()` I used for inspiration :sweat_smile: